pythoness/pythoness_module.py

```diff
@@ -332,7 +348,31 @@ def wrapper(*args, **kwargs):
                 if tests:
                     for t in tests:
                         try:
-                            if not eval(t):
+                            if isinstance(t, tuple):
+                                compiled_hypothesis_test = create_hypothesis_test(t)
+                                exec(compiled_hypothesis_test,globals())
+                            else:
+                                if not eval(t):
+                                    failing_tests.add(t)
+                        except AssertionError:
+                            #print(locals())
+                            exc_type, exc_value, exc_tb = sys.exc_info()
+                            tb = traceback.TracebackException(exc_type, exc_value, exc_tb)
+                            exception_info = tb.format_exception_only()
+                            line_number = 0
+                            falsifying_example = get_falsifying_example(exception_info)
+                            for exception_line in exception_info:
+                                print(str(line_number) + " "+ exception_line)
+                                line_number += 1
+
+                            print("Falsifying example is "+ falsifying_example)
```

Comment:
This is for debugging purposes, right? If so, guard with a debug flag or use the `logging` library and make it a log.debug message.